### PR TITLE
Migrate staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go structured logging

### PR DESCRIPTION
**Pull request by gaoguangze111**
In staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go

- Change log to structured logging

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Migrate some logs in staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go to structured logging.

**Which issue(s) this PR fixes**:

Fixes #

Ref:
- keps/sig-instrumentation/1602-structured-logging
- Structured Logging migration instructions

### CODE REVIEW

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go [hunk 1]
```diff
@@ -29,13 +29,13 @@ import (
 // indicating that the controller identified by controllerName is waiting for syncs, followed by
 // either a successful or failed sync.
 func WaitForCacheSync(controllerName string, stopCh <-chan struct{}, cacheSyncs ...cache.InformerSynced) bool {
-	klog.Infof("Waiting for caches to sync for %s controller", controllerName)
+	klog.InfoS("Waiting for caches to sync", "controller", controllerName)
```
Comment (shubheksha) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go:32]:
```suggestion
	klog.InfoS("Waiting for caches to sync", "controlleNamer", controllerName)
```

Ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go [hunk 2]
```diff
@@ -29,13 +29,13 @@ import (
...
 
 	if !cache.WaitForCacheSync(stopCh, cacheSyncs...) {
 		utilruntime.HandleError(fmt.Errorf("Unable to sync caches for %s controller", controllerName))
 		return false
 	}
 
-	klog.Infof("Caches are synced for %s controller", controllerName)
+	klog.InfoS("Caches are synced", "controller", controllerName)
```
Comment (shubheksha) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/cache.go:39]:
```suggestion
	klog.InfoS("Caches are synced", "controllerName", controllerName)
```
Ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments

**Review by shubheksha — CHANGES_REQUESTED** [review]

**Comment by yangjunmyfm192085** [comment]
May be this pr is repeated with #97595, you need keep only one?

**Pull request closed without merging**